src/olympia/access/acl.py

```diff
@@ -71,9 +71,7 @@ def check_collection_ownership(request, collection, require_owner=False):
     if not request.user.is_authenticated():
         return False
 
-    if action_allowed(request, amo.permissions.ADMIN):
-        return True
-    elif action_allowed(request, amo.permissions.COLLECTIONS_EDIT):
+    if action_allowed(request, amo.permissions.COLLECTIONS_EDIT):
```

Comment:
observation: we could rewrite this entire method into a single `return xxx and (yyy or zzz)` statement.  With debatable improvement on code readability.

Reply:
Yeah, it's not really any better IMHO. Maybe it would be with intermediate variables but I don't have the motivation to do that right now...